src/pages/HomePage/components/PopularDiets/PopularDiets.tsx

```diff
@@ -1,9 +1,97 @@
 import './PopularDiets.scss';
+import arrowleft from './arrow left.svg'
+import arrowright from './arrow right.svg'
+import Maskgroupslim from './Mask group slim.svg'
+import Maskgroupfit from './Mask group fit.svg'
+import Maskgroupvegan from './Mask group vegan.svg'
+import Maskgroupketo from './Mask group keto.svg'
 
 export const PopularDiets= () =>{
     return(
-        <>
-        <h1>PopularDiets</h1>
-        </>
+        <div className="PopularDiets narrow-container">
+        <p className="themost">The most popular diets</p>
+        <div className="arrows">
+            <button className="arrow1"><img className="picarrow1" src={arrowleft}/></button>
+            <button className="arrow2"><img className="picarrow2" src={arrowright}/></button>
+        </div>
+        <div className="fourdiets">
+            <div className="diet">
```

Comment:
1. create a Diet component
2. create an array in PopularDiets component with diet-objects
3. use diets.map(diet => <Diet ... />)
to render your Diet components